### PR TITLE
[WIP] Taught Godot how to redirect the texture locations when import_animat…

### PR DESCRIPTION
**Pull request by fire**
…ion_from_other_importer.

Found a problem where import_scene_from_other_importer doesn't give enough information to redirect .import files to the original location.

### CODE REVIEW

**Comment by akien-mga** [comment]
It would be better to find a way to get back to the original path directly by looking it up in the import remaps hashmap. There might already be an API for that, but best wait for @reduz to give input on that.

File: editor/import/editor_scene_importer_gltf.cpp [hunk 1]
```diff
@@ -1126,7 +1126,13 @@ Error EditorSceneImporterGLTF::_parse_images(GLTFState &state, const String &p_b
 				data_size = data.size();
 			} else {
 
-				uri = p_base_path.plus_file(uri).replace("\\", "/"); //fix for windows
+				String base_path;
+				
+				if (p_base_path.begins_with("res://.import")) {
+					base_path = p_original_base_path;
```
Comment (fire) [inline editor/import/editor_scene_importer_gltf.cpp]: Adding the a parameter is to enable this.

**Comment by fire** [comment]
There is two ways you can can call import scene from other importer:

1. call it from an existing location in the file system
1. use the external exporter then call the api with the exporter location and the original location. For example somewhere in .import and the original location.

The first one requires the associated data to be in the same place. This is the normal case.

The second case requires the .import scene to be relocated so that the associated data is at the original location.

**Comment by fire** [comment]
Updated.

**Comment by akien-mga** [comment]
Moving to 3.2 milestone as we're about to enter the beta stage and *release freeze* for Godot 3.1. Only bug fixes against issues of the 3.1 milestone (or enhancements to core features of the 3.1 roadmap) will be considered until 3.1-stable is released.

**Comment by fire** [comment]
My original case for using this was to import_scene_from_other_importer for FBX via GLTF2, but since that isn't my approach anymore, I'm not able to test this.

import_scene_from_other_importer is still broken.., but I'm not using it.

**Pull request closed without merging**